chromium/browser_type.go

```diff
@@ -371,15 +371,16 @@ func execute(
 	// can run into a data race.
 	err = cmd.Start()
 	if os.IsNotExist(err) {
-		return nil, nil, fmt.Errorf("does not exist: %s", path)
+		return nil, nil, fmt.Errorf("file does not exist: %s", path)
```

Comment:
Is this when it can't find the browser exec? I think it was ok as `does not exists`

Reply:
? 🤔 

```suggestion
		return nil, nil, fmt.Errorf("browser executable does not exist: %s", path)
```

Reply:
:thinking: But we don't know `path` refers to a browser executable at this point. `os.IsNotExist()` simply checks for the existance of a file. Whether that points to a browser executable and whether we can ultimately run it should be mentioned in a different error message IMO.